src/components/TopBar.js

```diff
@@ -230,143 +308,93 @@ export default _.flow(
           h(Clickable, {
             as: 'a',
             ...Utils.newTabLinkProps,
-            style: styles.nav.item,
+            style: { ...styles.nav.item, borderBottom: `1px solid ${colors.dark(0.55)}` },
             hover: { backgroundColor: colors.dark(0.55) },
             href: getConfig().jobManagerUrlRoot,
             onClick: () => this.hideNav()
           }, [
             div({ style: styles.nav.icon }, [
               icon('layers', { className: 'is-solid', size: 24 })
             ]),
-            'Your Jobs',
-            icon('pop-out', {
-              size: 12,
-              style: { marginLeft: '0.5rem' }
-            })
-          ]),
-          div({ style: { borderBottom: styles.nav.item.borderBottom, padding: '14px 0' } }, [
-            div({ style: { ...styles.nav.subItem, paddingLeft: 28 } }, [
-              div({ style: styles.nav.icon }, [
-                icon('library', { className: 'is-solid', size: 24 })
-              ]),
-              'Library'
-            ]),
-            librarySubItem('library-datasets', 'data-cluster', 'Datasets'),
-            librarySubItem('library-showcase', 'grid-chart', 'Showcase & Tutorials'),
-            librarySubItem('library-code', 'tools', 'Code & Tools')
+            'Job Manager'
           ]),
+          div({ style: { margin: '5rem' } }),
```

Comment:
This causes there to be a vertical scrollbar in the left nav if the window is too short (e.g. with dev tools open). I know it was discussed elsewhere but I also think this spacer is awkward. Was the intent that the parts below the spacer would be anchored to the bottom of the left nav panel?

Reply:
Hmm... scrollbar also appears if sections are open when the window is too short. I just want to avoid there ever being a scrollbar in this nav pane. If it's too tall for the window, the whole page should scroll instead. I don't know how to fix this.

Reply:
That would require a redo of the top level layout, and would have some problems of its own. You'd either need to show 'dead space' below the main content, or else have things resize on the fly, both of which tend to look awkward. I think the scroll bar is probably the best option.